src/main/webapp/src/app/login-signup-page/login-signup.component.html

```diff
@@ -10,3 +10,65 @@ <h1 class="greeting">Welcome To GrowPod</h1>
     </button>
   </div>
 </body>
+
+<ng-template #newUserForm>
+  <mat-card class="mat-elevation-z8 form">
+    <mat-card-title>
+      Tell us a little more about yourself!
+    </mat-card-title>
+
+    <mat-card-content>
+      <form [formGroup]="userGroup">
+        <mat-form-field appearance="fill">
+          <input
+            matInput
+            placeholder="Zipcode"
+            [(ngModel)]="zipCode"
+            formControlName="zipCodeValidator"
+          />
+          <mat-error
+            *ngIf="userGroup.controls.zipCodeValidator.hasError('required')"
+          >
+            Zipcode is required
+          </mat-error> </mat-form-field
+        ><br />
+
+        <mat-form-field appearance="fill">
+          <textarea
+            matInput
+            placeholder="Bio"
+            [(ngModel)]="bio"
+            formControlName="bioValidator"
+          ></textarea>
+          <mat-error
+            *ngIf="userGroup.controls.bioValidator.hasError('required')"
+          >
+            Biography is required
+          </mat-error> </mat-form-field
+        ><br />
+
+        <mat-form-field appearance="fill">
+          <mat-label>Are you joining or creating a garden?</mat-label>
+          <mat-select formControlName="gardenValidator">
+            <mat-option value="option">Joining</mat-option>
+            <mat-option value="option">Creating</mat-option>
+          </mat-select>
+          <mat-error
+            *ngIf="userGroup.controls.gardenValidator.hasError('required')"
+          >
+            This field is required
```

Comment:
nit: `This field is required` -> `Please select an option`.

Reply:
done